setup.cfg

```diff
@@ -85,7 +85,7 @@ install_requires =
     # The usage was added in #30596, seemingly only to override and improve the default error message.
     # Either revert that change or find another way, preferably without using connexion internals.
     # This limit can be removed after https://github.com/apache/airflow/issues/35234 is fixed
-    connexion[flask]>=2.10.0,<3.0
+    connexion[flask]>=2.10.0
```

Comment:
I would recommend bumping to 3.0 directly since it would probably be hard to write code that is completely compatible with both 2.X and 3.0.